README.md

```diff
@@ -24,14 +24,52 @@ GNU make requires GNU m4, automake, autoconf, Perl, and a C compiler that is gcc
 Perl requires a c89 compiler
 ncurses requires an ANSI C compiler 
 
-TL;DR to port zsh, first port:
- - m4
- - Perl
- - autoconf
- - automake
- - GNU make
- - ncurses
-You will likely want to port:
- - git
- - curl
- - techinfo
+## Order to Build _from scratch_
+
+If you want to build the tools from scratch and not use the binary pax files available, you will want 
+to tackle this in a particular order. 
+First, you need to have some tools installed on your system:
+
+### System Pre-reqs:
+
+ - gnu make 4.1: Download unsupported binary from Rocket
+ - xlclang 2.4.1: Download unsupported binary from IBM
+ - git: Download unsupported binary from Rocket
+ - curl: Download unsupported binary from Rocket
+ - gunzip: Download unsupported binary from Rocket
+
+### Recommended software:
+ - bash: Download unsupported binary from Rocket
+
+Both IBM and Rocket provide supported versions of the software above for a fee.
+
+Taking the defaults will mean there are less variables for you to configure. We recommend you structure your sandbox as follows:
+
+ - Have the root of your development file system be $HOME/zot (you will want to have several gigabytes of storage for use - we recommend at least 15GB)
```

Comment:
I changed the deps lookup path to $HOME/zopen here: https://github.com/ZOSOpenTools/utils/blob/main/bin/zopen-build#L105